src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteClient.java

```diff
@@ -66,6 +70,47 @@ public static MapRouletteClient instance(final MapRouletteConfiguration configur
         return null;
     }
 
+    /**
+     * This method writes challenge ids to a file.
+     *
+     * @param challengeId
+     *            challenge id of the newly created MapRoulette challenge.
+     * @param projectId
+     *            project id of the MapRoulette challenge.
+     */
+    public static void writeChallengeIdsToFile(final long challengeId, final long projectId)
+    {
+        outputPath.ifPresent(fileName ->
+        {
+            try
+            {
+                final BufferedWriter fileWriter = new BufferedWriter(
+                        new FileWriter(fileName, true));
+                fileWriter
+                        .append(String.format("project: %d;challenge: %d", projectId, challengeId));
+                fileWriter.newLine();
+                fileWriter.close();
+            }
+            catch (final IOException ioException)
+            {
+                logger.warn(
+                        "IOException occurred while writing project id {}, challenge id {} to the file {}",
```

Comment:
nit: we could print the stack trace or some detail about the exception for clarity

Reply:
Sure.

Reply:
Sonar doesn't like printstacktrace. However, i was logging the error in catch block.